src/libraries/System.Net.Sockets/tests/FunctionalTests/Connect.cs

```diff
@@ -82,6 +82,20 @@ public async Task Connect_MultipleIPAddresses_Success(IPAddress listenAt)
             }
         }
 
+        [OuterLoop("Slow on Windows")]
```

Comment:
I did something like 

```c#
            using (Socket listener = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
            {
                listener.Bind(new IPEndPoint(IPAddress.Loopback, 10000));

                Console.WriteLine("Connecting  {0}", DateTime.Now);
                var clientSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
                try
                {
                    await clientSocket.ConnectAsync(listener.LocalEndPoint);
                } 
                catch(Exception ex)
                {
                    Console.WriteLine("Connect failed {0} {1}", ex.Message, DateTime.Now);

                }
    }
```
and I get this on Windows 10
```
Connecting  6/19/2020 10:48:07 AM
Connect failed No connection could be made because the target machine actively refused it. 6/19/2020 10:48:09 AM
```
it seems like it takes 2s to fail. That does not seems to bad.